portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessAdminTaxonomyAPI/ExposeBatchActionForHeadlessAdminTaxonomyAPI.testcase

```diff
@@ -0,0 +1,60 @@
+@component-name = "portal-headless-frontend-infrastructure"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Headless Frontend Infrastructure";
+
+	setUp {
+		TestCase.setUpPortalInstanceNoSelenium();
+	}
+
+	tearDown {
+		JSONDepot.deleteDepot(
+			depotName = "Test Depot Name",
+			noSelenium = "true");
+	}
+
+	@disable-webdriver = "true"
+	@priority = "4"
+	test IncludeBatchEndpointsInActionBlockForAssetLibraryTaxonomyVocabulary {
+		property portal.acceptance = "true";
+
+		task ("Given an asset library is created") {
+			JSONDepot.addDepot(depotName = "Test Depot Name");
+		}
+
+		task ("When with get request and siteId to retrieve taxonomy vocabularies") {
+			var assetLibraryId = JSONGroupAPI._getDepotIdByName(
+				depotName = "Test Depot Name",
+				noSelenium = "true");
+
+			var response = TaxonomyVocabularyAPI.getTaxonomyVocabularyWithDifferentParameters(assetLibraryId = "${assetLibraryId}");
+		}
+
+		task ("Then I can see details of updateBatch, createBatch and deleteBatch inside of action block in response") {
+			TaxonomyVocabularyAPI.assertProperBatchInActionBlock(
+				assetLibraryId = "${assetLibraryId}",
+				batchActions = "createBatch,updateBatch,deleteBatch",
```

Comment:
Maybe we can remove the if statements and look directly for PUT, POST, and DELETE? I mean, in the JSON we'll look in the batch section anyways, right?
I would leave the task title as it is, but change the batchActions to the exact type. What do you think about it? 🤔